src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -34,133 +34,96 @@ kubectl config current-context
 
-    2. Add the New Relic Helm charts repo:
+We recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart for Kubernetes deployments, as it provides the easiest way to select, install, and upgrade individual components at your will.
 
-    ```
-    helm repo add newrelic https://helm-charts.newrelic.com
-    ```
+#### Installing and configuring `nri-bundle` with Helm
 
-    3. Make sure everything is configured properly in the chart by running the following command. This step uses the
-    `--dry-run` and `--debug` switches and therefore the agent isn't installed.
+1. Ensure you are using the appropriate context in the machine where you will run Helm and `kubectl`:
 
-    ```
-    helm upgrade --install <var>newrelic</var> newrelic/nri-bundle \
-    --dry-run \
-    --debug \
-    --set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
-    --set global.cluster=<var>K8S_CLUSTER_NAME</var> \
-    --set infrastructure.enabled=true \
-    --set prometheus.enabled=true \
-    --set webhook.enabled=true \
-    --set ksm.enabled=<var>false</var> \
-    --set kubeEvents.enabled=true \
-    --set logging.enabled=true
-    ```
+You can check the available contexts with:
 
-    You can flip the `ksm.enabled` flag to `true` if KSM isn't already present in your cluster.
-
-    4. Install the Kubernetes integration:
+```
+kubectl config get-contexts
+```
 
-    ```
-    helm upgrade --install <var>newrelic</var> newrelic/nri-bundle \
-    --set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
-    --set global.cluster=<var>K8S_CLUSTER_NAME</var> \
-    --set infrastructure.enabled=true \
-    --set prometheus.enabled=true \
-    --set webhook.enabled=true \
-    --set ksm.enabled=<var>false</var> \
-    --set kubeEvents.enabled=true \
-    --set logging.enabled=true
-    ```
+And switch to the desired context using:
 
-    5. Wait a few seconds, then check that the `DaemonSet` and pods have been created:
+```
+kc config use-context <var>CONTEXT_NAME</var>
+```
 
-    ```
-    kubectl get daemonsets,pods
-    ```
+2. Add the New Relic Helm charts repo:
 
-    6. Make sure you see a `DaemonSet`, and one pod per node for `newrelic-infrastructure`, as well as one `Deployment`
-    and one pod for any other component that you have enabled.
+```
+helm repo add newrelic https://helm-charts.newrelic.com
+```
 
-    ### Configuring `nri-bundle` with Helm
+3. Make sure everything is configured properly in the chart by running the following command. Notice that we are specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
 
-    #### Enabling and disabling subcharts
+```
+helm upgrade --install <mark>newrelic</mark> newrelic/nri-bundle \
```

Comment:
I am not sure it is clear that the wizard will populate the command automatically

Reply:
My idea was for this page to be a guide for the "manual installation" using helm. The automated installer should be self-documented.